File: app/src/main/java/com/example/user/uberuber/Remotes/IGoogleAPIService.java
package com.example.user.uberuber.Remotes;

import com.example.user.uberuber.Model.MyPlaces;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Url;

/**
 * Created by User on 7/4/2018.
 */

public interface IGoogleAPIService {
    @GET
    Call<MyPlaces> getNearByPlaces(@Url String url);
}

File: app/src/main/java/com/example/user/uberuber/Remo/IGoogleAPIService.kt
package com.example.user.uberuber.Remo

import com.example.user.uberuber.Models.MyPlaces
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

/**
 * Created by User on 17/4/2018.
 */
interface IGoogleAPIService {
    @GET
    fun getNearbyPlaces(@Url url:String): Call<MyPlaces>
}